doc/whats_new.rst

```diff
@@ -18,6 +18,7 @@ Changelog
 Bug
 ~~~
 
+- Add a copyfile_ctf function in mne_bids.utils to copy and rename CTF files during mne_bids.write_raw_bids, by `Romain Quentin`_ (`#162 <https://github.com/mne-tools/mne-bids/pull/162>`_)
```

Comment:
uff ... sorry for the last minute catch but you should use :func: so that the function is linked. See some examples below